### PR TITLE
bump AMO version to 1.2.0

### PR DESCRIPTION
**Pull request by HubertStefanski**
# Description
Bumping AMO version with new operator-sdk (0.15.2)
bug fix to : https://issues.redhat.com/browse/INTLY-8046

### CODE REVIEW

**Comment by HubertStefanski** [comment]
Ping @matskiv  @pb82

File: manifests/integreatly-monitoring/1.2.0/application-monitoring-operator.v1.2.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -2,27 +2,26 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
+    alm-examples: '[]'
     capabilities: Basic Install
     categories: Monitoring stack
     certified: "false"
-    containerImage: quay.io/integreatly/application-monitoring-operator:v1.1.6
+    containerImage: quay.io/integreatly/application-monitoring-operator:v1.1.5
```
Comment (matskiv) [inline manifests/integreatly-monitoring/1.2.0/application-monitoring-operator.v1.2.0.clusterserviceversion.yaml:9]:
```suggestion
    containerImage: quay.io/integreatly/application-monitoring-operator:v1.2.0
```

**Comment by matskiv** [comment]
/test e2e

Retrying. I am looking at the logs of the failed build in the mean time.

**Comment by HubertStefanski** [comment]
@david-martin

**Comment by matskiv** [comment]
/lgtm
🤞

**Comment by matskiv** [comment]
PROW problem with lease, rerunning
/test e2e

**Pull request merged**